### PR TITLE
OOP school library: add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by Tafloninno**
✔️  Setup linters for errors
✔️  Correct Gitflow
✔️  created person.rb file
✔️  created student.rb file
✔️  created teacher.rb file
✔️  Saved all classes into different files

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Student < Person
+  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+    super(id, age, parent_permission: parent_permission, name: name)
```
Comment (AmaduKamara) [inline student.rb]: [ Recommendation change ]

- [x] You could remove the id you are passing in the super() as it's not being received for this class. The id is auto-generated in the Person class and is not expected to be inputted when a student is created. It's okay to remove it from the super. 👍🏽

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Teacher < Person
+  def initialize(age, specialization, parent_permission: true, name: 'Unknown')
+    super(id, age, parent_permission: parent_permission, name: name)
+    @specialization = specialization
```
Comment (AmaduKamara) [inline teacher.rb]: [ Recommendation change ]

- [x] You could remove the id you are passing in the super() as it's not being received for this class. The id is auto-generated in the Person class and is not expected to be inputted when a teacher is created. It's okay to remove it from the super. 👍🏽

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @Tafloninno,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Created classes are required 👍🏽

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: teacher.rb (hunk 2 above)
Comment (Tafloninno) [reply teacher.rb]: alright thank you @AmaduKamara

**Pull request merged**